flink-formats/flink-avro-confluent-registry/pom.xml

```diff
@@ -33,7 +33,7 @@ under the License.
 
 	<properties>
 		<kafka.version>2.4.1</kafka.version>
-		<confluent.version>5.4.2</confluent.version>
+		<confluent.version>5.5.1</confluent.version>
 	</properties>
```

Comment:
Upgrade to new version client so that we can fetch for format type from the `SchemaMetadata`.